libmscore/sym.cpp

```diff
@@ -6675,6 +6670,17 @@ void ScoreFont::scanUserFonts(const QString& path, bool system)
 
       _allScoreFonts = _builtinScoreFonts;
       _allScoreFonts << _userScoreFonts << _systemScoreFonts;
+
+      // Include external and internal score fonts into QFontDatabase
+      for (auto& f : _allScoreFonts) {
```

Comment:
Before I was doing _userScoreFonts but now _allScoreFonts because it looked like they were already loaded in there, but they weren't. Only the "text" fonts were loaded in, where I put that comment above

Reply:
Does it also fix your original problem?

Reply:
The code as it was (and is here) hasn't had the crash so far, so I think so. I'll report if I ever experience it again